src/themes/eLife/styles.css

```diff
@@ -1,396 +1,39 @@
-@import '../../common/styles/common.css';
-@import 'prismjs/themes/prism.css';
+/* Ideally use import as below, but currently encounters build issues, */
+/* and inability to install as NPM package due to missing package version number. */
+/* @import 'eLife-pattern-library/assets/sass/build.scss'; */
+/* due to the above issues we import and inline the built CSS file from the web */
+@import 'https://elifesciences.org/assets/patterns/css/all.0565330b.css';
```

Comment:
This will not track evolving eLife styles due to the cachebusting we use: for `all.[some-alphanumeric].css`, the `[some-alphanumeric]` part is different every time the file is changed.

Dockerhub houses [the current asset set](https://hub.docker.com/r/elifesciences/pattern-library_assets) for the patterns driving the eLife site, including the css and images controlled by the pattern library. Because it is automatically updated by our CI whenever a branch is merged into master of https://github.com/elifesciences/pattern-library, it's always up to date. Might be possible to grab this and extract what's needed?